plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -422,6 +422,14 @@ func ClusterRoles() []rbac.ClusterRole {
 				eventsRule(),
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "system:cloud-provider"},
+			Rules: []rbac.PolicyRule{
+				rbac.NewRule("get", "list", "watch").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
+				rbac.NewRule("patch", "update").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
```

Comment:
Is `update` required? I only see `patch` being used in `controller.AddOrUpdateTaintOnNode`.